crates/prelude/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.6.0-git
+
+### Major
+
+- Change `button::Listener::new()` to never panic and return an error instead.
+
 ## 0.5.1-git
```

Comment:
The 0.6.0-git section is not a new one, but a major bump of the current -git version. I'll apply the change and merge.

```suggestion
- Change `button::Listener::new()` to never panic and return an error instead
```